cloud/disk_manager/internal/pkg/dataplane/url/vhd/header.go

```diff
@@ -6,26 +6,27 @@ import "github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/dataplane/ur
 
 const (
 	// The dynamic disk header should appear on a sector (512-byte) boundary.
-	headerOffset = uint64(512)
-
-	headerCookie = "cxsparse"
+	headerOffset  = uint64(512)
+	headerCookie  = "cxsparse"
+	batOffset     = uint64(1536)
```

Comment:
What is "bat"?

Reply:
it is "block allocation table"

it was added here 
https://github.com/ydb-platform/nbs/commit/738a41e4b37e2a4d3c61a4eaa74accbc6fa781e0

Reply:
Need comment about that please